src/clr-base/sass-build.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const Fiber = require('fibers');
+const gulp = require('gulp');
+const sass = require('gulp-sass');
+const rename = require('gulp-rename');
+const gap = require('gulp-append-prepend');
+
+sass.compiler = require('sass');
+
+gulp.task('sass', () =>
+  gulp
+    .src('./components/**/*.scss', { base: './' })
+    .pipe(sass({ fiber: Fiber, outputStyle: 'compressed' }).on('error', sass.logError))
+    .pipe(rename(path => (path.extname = '.css.ts')))
+    .pipe(gap.prependText("/* tslint:disable */import { css } from 'lit-element';\n\nexport const styles = css`"))
+    .pipe(gap.appendText('`;'))
+    .pipe(gulp.dest(file => file.base))
+);
+
+gulp.task('sass:watch', () => gulp.watch('./**/*.scss', gulp.series('sass')));
```

Comment:
I didnt really care for this solution that much but until I can get the rollup plugin working this will get us a good head start.

Reply:
Also I really want to get rid of this so we don't need the gulp dependency.

Reply:
Fixed, was able to use a plugin from the polymer project that remove the need for gulp